packages/office-ui-fabric-react/src/components/Separator/Separator.base.tsx

```diff
@@ -13,19 +13,19 @@ export class SeparatorBase extends BaseComponent<ISeparatorProps, {}> {
   }
 
   public render(): JSX.Element {
```

Comment:
Just a suggestion...looks like this component can be a functional component and not need an overhead of a class. Plus David is slowly removing the BaseComponent from all the components.

Reply:
I agree.. there's no state here so we shouldn't need a class. Also, the constructor doesn't seem necessary.

Reply:
Good catch all!